icepyx/core/granules.py

```diff
@@ -361,7 +370,7 @@ def place_order(
             request.raise_for_status()
             esir_root = ET.fromstring(request.content)
             if verbose is True:
-                print("Order request URL: ", requests.utils.unquote(request.url))
+                print("Order request URL: ", unquote(request.url))
```

Comment:
We now import `requests.compat.unquote` - is this just an alias?

Reply:
isn't not an alias, i.e. we're not using `as` in the import; we're importing `unquote` with its original name:

```
from requests.compat import unquote
```

This change was prompted by the typechecker. I didn't dig deep in to it since the change seemed fine and the queue of typechecker errors was pretty long :)

Reply:
The part that was confusing to me was that we went from `requests.utils.unquote` to `requests.compat.unquote`. They both lead to a function called `unquote` but in two different places, so I wanted to make sure they were doing the same thing. I was wondering if `requests.compat.unquote` was an alias for `requests.utils.unquote`.

Reply:
Ah. Sorry, my mistake! `requests` is just re-importing the `unquote` function in the `utils` module; it's original source is the `compat` module:

https://github.com/psf/requests/blob/f12ccbef6d6b95564da8d22e280d28c39d53f0e9/src/requests/utils.py#L44-L52

Reply:
Gotcha - that totally makes sense to switch then!